morpheus/messages/multi_inference_message.py

```diff
@@ -18,14 +18,14 @@
 
 import cupy as cp
 
-import morpheus._lib.messages as neom
+import morpheus._lib.messages as _messages
```

Comment:
Great readability improvement.

Reply:
Ya, this was an artifact. It used to be `neo.message` so `neom` made sense.